### ISSUE TITLE
Sorting options for missing fields

### ISSUE THREAD

**Issue by kpsherva** [inveniosoftware/invenio-app-rdm#2030]
- We currently (Zenodo-legacy) have:
  - Most viewed (moved to separate issue)
  - Best match
  - Most recent (default option when no search query)
  - Publication date
  - Conference session
  - Journal
  - Version (used by the record landing page "view all versions" link)
- Besides the sorting option there's also a "ascending/descending" dropdown available
- All of the sort options take into account safelisting (prior to the actual sorting)

### Prerequisites

- Metadata completeness (for the conference and journal options) [#2019](https://github.com/inveniosoftware/invenio-app-rdm/issues/2019)

### Next steps

1. Review ascending/descending behaviour
2. Review configuration for RDM records

### Current

![Image](https://user-images.githubusercontent.com/1814661/180229029-f9efa656-98d5-4804-a8f3-a9c20a19c3da.png)

**Comment by kpsherva** [comment]
there is some more context for the "Most viewed" sort, please contact me before starting

**Comment by ppanero** [comment]
- `ascending/descending` has been "deprecated" 
- `publication_date`, we currently have "oldest" and "newest" for creation date. Suggestion add two more: "newest publication data" and "oldest publication date" (naming accepting suggestions)
- TBC
